dialect/sql/schema/atlas.go

```diff
@@ -264,6 +264,13 @@ func WithFormatter(fmt migrate.Formatter) MigrateOption {
 	}
 }
 
+// EnableSumFile instructs atlas to generate a migration directory integrity sum file as well.
+func EnableSumFile() MigrateOption {
```

Comment:
```suggestion
func WithSumFile() MigrateOption {
```

Maybe use `With` to stay consistent with other `MigrateOption`s.